tutorials/2_Gazebo_and_ROS.ipynb

```diff
@@ -0,0 +1,1330 @@
+    "    <li>How to add a plugin to the model to allow ROS commands to be sent to the robot</li>\n",
+    "    <li>How to move the robot with ROS messages</li>\n",
+    "</ul>    \n",
+    "\n",
+    "<p>A differential wheeled robot is a mobile robot whose movement is based on two separately driven wheels placed on either side of the robot body. It can thus change its direction by varying the relative rate of rotation of its wheels and hence does not require an additional steering motion.</p>\n",
+    "<p><img src='http://42bots.com/wp-content/uploads/2014/01/Differential-Steering-Tutorials-42bots2.png' width='600px'/></p>\n",
+    "\n",
+    "\n",
+    "<p>First we need to create our diff_wheeled_robot_description and diff_wheeled_robot_gazebo package in our catkin workspace folder</p>\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "cd catkin_ws/src"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "catkin_create_pkg diff_wheeled_robot_description\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "catkin_create_pkg diff_wheeled_robot_gazebo\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h2>Creating your Gazebo Launch File</h2>\n",
+    "\n",
+    "<p>Let’s start with the gazebo package, go in there and create the following subfolders:</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "roscd diff_wheeled_robot_gazebo \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mkdir launch \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Create a new file:</p>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "gedit diff_wheeled_gazebo.launch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Add an empty world and try it </p>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<launch>\n",
+    "  <include file=\"$(find gazebo_ros)/launch/empty_world.launch\">\n",
+    "  </include>\n",
+    "</launch>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will include an empty world launch file from gazebo_ros package which will launch gazebo with nothing inside. Open a new terminal and launch the file."
```

Comment:
You mention that you will include an empty world launch file, but don't give instructions. The launch file at this point is effectively empty.